apps/etterlatte-behandling/src/main/kotlin/behandling/aktivitetsplikt/AktivitetspliktRoute.kt

```diff
@@ -47,20 +46,18 @@ internal fun Route.aktivitetspliktRoutes(aktivitetspliktService: Aktivitetsplikt
 
         post {
             kunSkrivetilgang {
-                hentNavidentFraToken { navIdent ->
-                    val oppfolging = call.receive<OpprettAktivitetspliktOppfolging>()
-
-                    try {
-                        val result =
-                            aktivitetspliktService.lagreAktivitetspliktOppfolging(
-                                behandlingId,
-                                oppfolging,
-                                navIdent,
-                            )
-                        call.respond(result)
-                    } catch (e: TilstandException.UgyldigTilstand) {
-                        call.respond(HttpStatusCode.BadRequest, "Kunne ikke endre på feltet")
-                    }
+                val oppfolging = call.receive<OpprettAktivitetspliktOppfolging>()
+
+                try {
+                    val result =
+                        aktivitetspliktService.lagreAktivitetspliktOppfolging(
+                            behandlingId,
+                            oppfolging,
+                            brukerTokenInfo.ident(),
```

Comment:
For behandlingsappen sin del så hvis dette er en systembruker så vil man nå kun få "EY".
Spørsmålet er hva som er mest riktig er her av det og:
```
            .hentTokenClaims(AZURE_ISSUER)
            ?.getStringClaim("azp_name") 
```

Som brukes i `class SystemUser`. Og hvordan burde kontekst innholdet som settes matche med det som ligger på BrukerTokenInfo?

Reply:
Ah jeg var litt rask på avtrekkeren der, ser at BrukerTokenInfo ideelt sett returnerer `azp_name`.
Så da er det ikke like relevant.